src/Security/SecProtocolOptions.cs

```diff
@@ -0,0 +1,203 @@
+
+		[TV (12,0), Mac (10,14), iOS (12,0)]
+		public void SetTlsOcspEnabled (bool ocspEnabled) => sec_protocol_options_set_tls_ocsp_enabled (GetHandle (), (byte)(ocspEnabled ? 1 : 0));
+
+		[TV (12,0), Mac (10,14), iOS (12,0)]
+		[DllImport (Constants.SecurityLibrary)]
+		static extern void sec_protocol_options_set_tls_sct_enabled (IntPtr handle, byte sctEnabled);
+
+		[TV (12,0), Mac (10,14), iOS (12,0)]
+		public void SetTlsSignCertificateTimestampEnabled (bool sctEnabled) => sec_protocol_options_set_tls_sct_enabled (GetHandle (), (byte)(sctEnabled ? 1 : 0));
+
+		[TV (12,0), Mac (10,14), iOS (12,0)]
+		[DllImport (Constants.SecurityLibrary)]
+		static extern void sec_protocol_options_set_tls_renegotiation_enabled (IntPtr handle, byte renegotiationEnabled);
+
+		[TV (12,0), Mac (10,14), iOS (12,0)]
+		public void SetTlsRenegotiationEnabled (bool renegotiationEnabled) => sec_protocol_options_set_tls_renegotiation_enabled (GetHandle (), (byte)(renegotiationEnabled ? 1 : 0));
+
+		[TV (12,0), Mac (10,14), iOS (12,0)]
+		[DllImport (Constants.SecurityLibrary)]
+		static extern void sec_protocol_options_set_peer_authentication_required (IntPtr handle, byte peerAuthenticationRequired);
+
+		[TV (12,0), Mac (10,14), iOS (12,0)]
+		public void SetPeerAuthenticationRequired (bool peerAuthenticationRequired) => sec_protocol_options_set_peer_authentication_required (GetHandle (), (byte)(peerAuthenticationRequired ? 1 : 0));
+
+		[TV (12,0), Mac (10,14), iOS (12,0)]
+		[DllImport (Constants.SecurityLibrary)]
+		static extern void sec_protocol_options_set_key_update_block(sec_protocol_options_t options, IntPtr key_update_block, dispatch_queue_t key_update_queue);
+
+		[TV (12,0), Mac (10,14), iOS (12,0)]
+		[BindingImpl (BindingImplOptions.Optimizable)]
+		public void SetKeyUpdateCallback (SecProtocolKeyUpdate keyUpdate, DispatchQueue keyUpdateQueue)
+		{
+			if (keyUpdate == null)
+				throw new ArgumentNullException(nameof(keyUpdate));
+			if (keyUpdateQueue == null)
+				throw new ArgumentNullException(nameof(keyUpdateQueue));
+			unsafe {
+                                BlockLiteral *block_ptr_handler;
+                                BlockLiteral block_handler;
+                                block_handler = new BlockLiteral ();
+                                block_ptr_handler = &block_handler;
+                                block_handler.SetupBlockUnsafe (Trampolines.SDSecProtocolKeyUpdate.Handler, keyUpdate);
+
+                                sec_protocol_options_set_key_update_block (handle, (IntPtr)((void*) block_ptr_handler), keyUpdateQueue.Handle);
+                                block_ptr_handler->CleanupBlock ();
+			}
+		}
+
+		[TV (12,0), Mac (10,14), iOS (12,0)]
+		[DllImport (Constants.SecurityLibrary)]
+		static extern void sec_protocol_options_set_challenge_block(sec_protocol_options_t options, IntPtr challenge_block, dispatch_queue_t challenge_queue);
```

Comment:
better comment this (and the next one), since they are not publicly exposed. That way xtro can report them as missing/ignored

Reply:
got it